test/spec/modules/teadsBidAdapter_spec.js

```diff
@@ -196,7 +197,50 @@ describe('teadsBidAdapter', function() {
       expect(payload.gdpr_iab).to.exist;
       expect(payload.gdpr_iab.consent).to.equal(consentString);
       expect(payload.gdpr_iab.status).to.equal(0);
-    })
+    });
+
+    it('should use good mediaTypes video playerSizes', function() {
+      const mediaTypesPlayerSize = {
```

Comment:
Also just want to confirm:

If a publisher requests a multi-format adUnit with banner and video, teads will default to Video and try to get the sizes from the video object.

Just want to confirm this is intended and there is no logic necessary for teads' approach to multiformat requests.

From a glance it does not look like mediaType is passed into the Teads Ad request anyways, but want to be sure that in this scenario, the behavior is as expected!

Reply:
ex:

```
{
   mediaTypes: {
      video: {
         playerSize: [400, 600]
      }
      banner: {
         sizes: [[300, 250], [728, 90]]
      }
}
```

Some adapters do a multi-format call with both, some default to just video, some to banner.

Reply:
Very good point! Teads does offer some special formats like parallax ads, which may come with another size. Should `video` and `banner` sizes simply be concatenated?

Reply:
There is only one kind of response from Teads. It means, banner or video will be the same for us. We check if it's video juste to get sizes properly. 
Looks ok for us 😀 
Thanks !

Reply:
That is hard to say without knowing the Teads Ad Exchange or what they tell customers about their implementation.

I think coming up with what the Teads approach to multi-format adUnits is would be a good start.

Maybe concatenating is the solution, maybe having one of the mediaTypes be higher priority (as video is in this current implementation)

Not sure!

Reply:
Good point, 
we prefer receive all sizes, so we will concatenate. I'll push the update soon. Thanks

Reply:
Done. :) Don't hesitate to ask if you find something weird.
Thanks !